packages/vscode-ext/package.json

```diff
@@ -18,10 +18,29 @@
     "Visualization"
   ],
   "activationEvents": [
-    "onCustomEditor:squiggle.wysiwyg"
+    "onCustomEditor:squiggle.wysiwyg",
+    "onCommand:squiggle.preview"
   ],
   "main": "./out/extension.js",
   "contributes": {
+    "languages": [
+      {
+        "id": "squiggle",
+        "extensions": [
+          ".squiggle"
```

Comment:
also .squiggleU, maybe?

Reply:
It won't work in preview, since SquigglePlayground can't handle it. But it might be useful for syntax highlighting. Uh, I can just declare two different languages...